patches/@mauron85+react-native-background-geolocation+0.6.3.patch

```diff
@@ -1,13 +1,886 @@
+diff --git a/node_modules/@mauron85/react-native-background-geolocation/android/common/VERSIONS.gradle b/node_modules/@mauron85/react-native-background-geolocation/android/common/VERSIONS.gradle
+index 61f0865..9f9576f 100644
+--- a/node_modules/@mauron85/react-native-background-geolocation/android/common/VERSIONS.gradle
++++ b/node_modules/@mauron85/react-native-background-geolocation/android/common/VERSIONS.gradle
+@@ -146,5 +146,6 @@ ext {
+         logback: [group: 'com.github.tony19', name: 'logback-android', version: '1.1.1-9'],
+         slf4j: [group: 'org.slf4j', name: 'slf4j-api', version: '1.7.21'],
+         promise: [group: 'com.github.jparkie', name: 'promise', version: '1.0.3'],
++        commonsio: [group: 'commons-io', name:'commons-io', version: '2.6'],
```

Comment:
Can you do the patch without jetifier changes so it's smaller/cleaner?.. there'a a lot going in here

@troach-sf knows how! because I asked the same of him